.gitignore

```diff
@@ -1,5 +1,8 @@
 /book-result/
 /pillarPostExport.sh
+.DS_Store
+BasicClasses/.DS_Store
+Collections/.DS_Store
```

Comment:
Hi @QDucasse ,
I think .gitignore accept this syntax: `**/.DS_Store`.  Like that, it will ignore *all* the .DS_Store because `**` means at any sublevel.